### ISSUE TITLE
There were no symbols found Error

### ISSUE THREAD

**Issue by Axent96** [maglnet/ComposerRequireChecker#396]
Just installed this and tried it, but I get the following Error:

ComposerRequireChecker 4.5.0@5696f5eeac91838cfcb01ef1ce9709f18f2bbbea

In CheckCommand.php line 170:

  There were no symbols found, please check your configuration.

check [--config-file CONFIG-FILE] [--ignore-parse-errors] [--output OUTPUT] [--] [<composer-json>]

I have tried it with a config file. Same result.
php v8.1 && php v8.2

**Comment by maglnet** [comment]
Hi, this tool tries to find your source files by the definitions within your composer.json file or by defining additional files to scan within the config file.

To analyze this issue we need more information about your project setup, composer.json / autoload section.

**Comment by Ocramius** [comment]
Potential resolution here could be to improve the error message, perhaps? :thinking:

**Comment by maglnet** [comment]
> Potential resolution here could be to improve the error message, perhaps? 🤔

Yes maybe we should show the analysed files/paths/autoload definitions.

**Comment by fabioknoedt** [comment]
@maglnet yes, please!
